pact/Pact/Core/IR/Term.hs

```diff
@@ -265,23 +266,22 @@ findDefInModule :: Text -> Module name ty b i -> Maybe (Def name ty b i)
 findDefInModule defnName targetModule =
   find ((==) defnName . defName) (_mDefs targetModule)
 
-defKind :: Def name Type b i -> DefKind
-defKind = \case
+defKind :: ModuleName -> Def name Type b i -> DefKind
+defKind mn = \case
```

Comment:
this is a little cheeky internally :). That said: `defKind` is always called when the module name is in-scope, so this is fine.